2021/ml4co-competition/index.html

```diff
@@ -192,15 +192,15 @@ <h2>Challenges</h2>
 				<ul class="features">
 					<li>
 						<h3><a href="#primal-task" class="button">Primal task</a></h3>
-						<p>Produce feasible solutions, in order to minimize the <em>primal integral</em> over time.</p>
+						<p>Produce feasible solutions, minimizing the <em>primal integral</em> over time.</p>
```

Comment:
I prefer the previous phrasing. Here it sounds that producing feasible solutions, in whichever way, minimizes the primal integral.

Reply:
The one I proposed is much more English sounded ;)

Reply:
A German and a French debate about the best way to write English. Sounds like the start of a great Joke :P

Reply:
Haha.